src/snowflake/cli/plugins/nativeapp/v2_conversions/v2_to_v1_decorator.py

```diff
@@ -106,10 +94,9 @@ def _pdf_v2_to_v1(v2_definition: DefinitionV20) -> DefinitionV11:
             "distribution"
         ] = app_package_definition.distribution
     if app_package_definition.meta and app_package_definition.meta.post_deploy:
-        pdfv1["native_app"]["package"]["scripts"] = [
-            _convert_v2_post_deploy_hook_to_v1_scripts(s)
-            for s in app_package_definition.meta.post_deploy
-        ]
+        pdfv1["native_app"]["package"][
+            "post_deploy"
+        ] = app_package_definition.meta.post_deploy
```

Comment:
Shouldn't we convert both `scripts` and `post_deploy`? Users can still have `native_app.package.scripts` in their definition v1 file.

Reply:
We are converting v2 to v1 here, so they can't have native_app.package.scripts in v2, or am I missing something?

Reply:
Ah sorry, you're right, I got it backwards.